src/Components/Results/Results.css

```
@@ -1,3 +1,38 @@
 .results-loading-container {
   height: 80vh;
 }
+
+.results-tab-container {
+  background-color: var(--primary-color);
+  padding-top: 0.75rem;
+}
+
+.results-tab-container > :first-child {
+  padding-left: 0.75rem;
+}
+
+.results-tab-container > :nth-child(2) {
+  padding-right: 0.75rem;
+}
```

You can just put the padding on the container. Also, for future reference, there is a `last-child` selector.
```suggestion
.results-tab-container {
  background-color: var(--primary-color);
  padding: 0.75rem 0.75rem 0 0.75rem;
}
```